radioactivedecay/inventory.py

```diff
@@ -200,49 +454,55 @@ def wrapper(*args, **kw):
 
 class Inventory:
     """
-    ``Inventory`` instances store a dictionary of radionuclides and associated activities, and a
+    ``Inventory`` instances store a dictionary of radionuclides and associated numbers, and a
     ``DecayData`` instance of radioactive decay data.
 
     Parameters
     ----------
     contents : dict
-        Dictionary containing radionuclide strings or Radionuclide objects as keys and activities
+        Dictionary containing radionuclide strings or Radionuclide objects as keys and numbers
         as values.
+    input_type : str, optional
+        Type of input (masses, numbers, activities; activities is default)
     check : bool, optional
         Check for the validity of contents (default is True).
     data : DecayData, optional
         Decay dataset (default is the ICRP-107 dataset).
 
     Attributes
     ----------
-    contents : dict
-        Dictionary containing radionuclide strings as keys and activities as values. Radionuclides
+    numbers : dict
+        Dictionary containing radionuclide strings as keys and numbers as values. Radionuclides
         in contents are sorted alphabetically.
     data : DecayData
         Decay dataset.
 
     Examples
     --------
     >>> rd.Inventory({'Tc-99m': 2.3, 'I-123': 5.8})
-    Inventory: {'I-123': 5.8, 'Tc-99m': 2.3}, decay dataset: icrp107
+    Inventory: {'I-123': 399738.47946141585, 'Tc-99m': 71852.27235544211}, decay dataset: icrp107
```

Comment:
Here so the API has change, but I think it would be better to keep it as it is, i.e. so it still reported activities. The way to do this is to edit the `__repr__()` method to call `self.activities()`. Maybe also change the string to "Inventory activities:" so it is explicit about what it is telling you.